doc-Installing_on_OpenShift_Container_Platform/topics/post-installation.adoc

```diff
@@ -145,6 +145,133 @@ $ oc create -f cfme-restore-job.yaml
 
 After the restore job is complete, you can scale the pods back up.
 
+
+
+[[updating-pods]]
+=== Updating {product-title_short}
+
+To apply package updates to a {product-title_short} appliance running on OpenShift Container Platform, delete the pods and redeploy them using updated {product-title_short} container images. If you have modified the template, this requires additional steps in xref:modified-template-update[].
```

Comment:
This says "If **you** have modified the template" .. That seems a bit confusing. The condition where a user would need to deploy an updated template is when we release a template with changes. It doesn't really have anything to do with if the user made modifications.

Reply:
I've updated the wording, hopefully it makes more sense now.